app/models/user.rb

```diff
@@ -48,6 +48,10 @@ def account_id
     @account_id ||= account&.id
   end
 
+  def initial_sign_in
+    user_account ? user_account.created_at.iso8601 : ""
```

Comment:
It's fine if you want to use the `iso8601` to format the string, but I think it makes more sense for that specifically in the serializer itself. So I would leave that part off of this function, but put it in the `users/profile.rb` file when you are defining the serialized field

Reply:
Cool it looks like I don't need to apply the `iso8601` function, maybe because I did not rename the field?  Not sure.  In any case, it works!  Maybe it worked before and I just added unneeded complexity